src/home-assistant.html

```diff
@@ -122,10 +122,10 @@
   loadBackendTranslations() {
     if (!this.hass.language) return;
```

Comment:
I think you need to change this to `selectedLanguage` too. See https://github.com/home-assistant/home-assistant/issues/13035, we're requesting `/api/translations/null` on startup.

```js
if (!this.hass. selectedLanguage) return;
```